hw/top_earlgrey/fpv/top_earlgrey_dedicated_fpv_cfgs.hjson

```diff
@@ -0,0 +1,14 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+
+  // This is the master cfg hjson for FPV. It imports ALL individual FPV
+  // dedicated cfgs of the IPs. This enables to run them all as a regression in one shot.
+  name: top_earlgrey_dedicated_ip_batch_fpv
+
+  import_cfgs: [// common server configuration for results upload
+                "{proj_root}/hw/data/common_project_cfg.hjson"
+                "{proj_root}/hw/formal/data/dedicated_fpv_cfgs.hjson"]
+  use_cfgs:    []
```

Comment:
This should not be needed.

Reply:
I guess this would be the first issue we encounter - If I take it away, I will get this error.. 
`ERROR: [FlowCfg] Key "use_cfgs" encountered in a non-master cfg file list "/usr/local/google/home/chencindy/opentitan/hw/top_earlgrey/fpv/top_earlgrey_dedicated_fpv_cfgs.hjson"`
I can work on fixing this in a separate PR. If the solution is not desired, I can revert back the code and not import master cfgs.